src/Core/Grid/Definition/Factory/CmsPageCategoryDefinitionFactory.php

```diff
@@ -193,6 +192,8 @@ protected function getFilters()
             $actionsTypeOptions['redirect_route_params'] = [
                 'id_cms_category' => $this->cmsCategoryParentId,
             ];
+
+            $actionsTypeOptions['reset_route_params']['id_cms_category'] = $this->cmsCategoryParentId;
```

Comment:
Is this needed for `admin_common_reset_search_by_filter_id` (this reset route supposedly only needs `filterId`)

Reply:
need to pass a parameter here in my action to keep the whole page on the same level after the action is applied :)